packages/geo-diary-v2/src/components/ui/appointment-list/appointment-list.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { AppointmentModel, ListItemModel } from '@reapit/foundations-ts-definitions'
+import { Tile, Button, IconList, getTime } from '@reapit/elements'
+import { FaClock, FaStreetView, FaAddressCard } from 'react-icons/fa'
+import { ExtendedAppointmentModel } from '@/types/global'
+
+export const renderFooterItems = () => [
+  <>
+    <Button type="submit" variant="primary" onClick={() => null} disabled={false} loading={false} fullWidth={false}>
```

Comment:
Defining `variant="primary"` here is unnecessary because the default variant of a button was primary already